tests/dft-bench-interaction/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+include(TestingMacros)
+
+add_regression_test(dft-bench-interaction "psi;dft;scf")
+set_tests_properties(dft-bench-interaction PROPERTIES TIMEOUT 3000) 
```

Comment:
If this test takes a long time, you can set its [`COST`](https://cmake.org/cmake/help/v3.2/prop_test/COST.html) so that it will start first.

Reply:
nifty, will do! that's better than playing games with alpha order and `LIST` order.

Reply:
Indeed! CTest calculates the `COST` property on first run (you may have noticed that longer tests start first when you re-run it).